jaeger-production-template.yml

```diff
@@ -12,6 +12,10 @@
 # the License.
 #
 
+#
+# Check https://github.com/jaegertracing/jaeger/releases for the latest version to use
```

Comment:
can you please remove this?

Reply:
This should rather link to dockerhub but I think everybody knows what registry it is using and where to find more info